accepted/future-releases/horizontal-inference/feature-specification.md

```diff
@@ -246,14 +258,16 @@ Performing type inference on an invocation consists of the following steps:
 
    * Horizontal inference: if generic inference is needed, and this is not the
      last stage, use all the type constraints gathered so far to find the
-     constraint solution for the target function's type variables, considering
-     bounds.  This produces an updated preliminary mapping of type parameters to
-     type schemas.
+     constraint solution for the target function's type variables, using the
+     mapping from the most recent previous execution of either this step or step
+     3 as the "previous mapping".  This produces an updated preliminary mapping
```

Comment:
I would propose uniformly using "partial solution" here instead of "mapping".  That is, talking about the "previous partial solution" and "produces an updated partial solution".  WDYT?